torchgen/model.py

```diff
@@ -788,9 +796,42 @@ def __post_init__(self) -> None:
                     f"that don't have matching signatures: {test_sig} != {f.func.signature()}"
                 )
         assert self.functional.func.kind() == SchemaKind.functional
+        assert (
+            not self.functional.is_view_op
+        ), "View operator shouldn't be grouped into NativeFunctionsGroup objects. Did you add an out= variant to a view operator?"
```

Comment:
I guess the error could be a little more directive here. Adding an out= variant for a view op is just Wrong(TM) and the error message should say so and ask the dev to ask for help